zhaquirks/xiaomi/aqara/sensor_switch_aq3.py

```diff
@@ -167,3 +181,59 @@ class SwitchAQ3B(XiaomiCustomDevice):
         (LONG_PRESS, LONG_PRESS): {COMMAND: COMMAND_HOLD},
         (LONG_RELEASE, LONG_RELEASE): {COMMAND: COMMAND_RELEASE},
     }
+
+
+class SwitchAQ3B2(XiaomiCustomDevice):
+    """Aqara button device - alternate version."""
+
+    signature = {
+        # <SimpleDescriptor endpoint=1 profile=260 device_type=259
+        # device_version=1
+        # input_clusters=[0, 1, 3]
+        # output_clusters=[3, 6, 19]>
+        MODELS_INFO: [(LUMI, "lumi.remote.b1acn02")],
+        ENDPOINTS: {
+            1: {
+                PROFILE_ID: zha.PROFILE_ID,
+                DEVICE_TYPE: BUTTON_DEVICE_TYPE_B,
+                INPUT_CLUSTERS: [
+                    Basic.cluster_id,
+                    XiaomiPowerConfiguration.cluster_id,
+                    Identify.cluster_id,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    OnOff.cluster_id,
+                    Ota.cluster_id,
+                ],
+            }
+        },
+    }
+    replacement = {
+        SKIP_CONFIGURATION: True,
+        ENDPOINTS: {
+            1: {
+                DEVICE_TYPE: zha.DeviceType.REMOTE_CONTROL,
+                INPUT_CLUSTERS: [
+                    BasicCluster,
+                    XiaomiPowerConfiguration,
+                    Identify.cluster_id,
+                    MultistateInputCluster,
+                    OppleCluster,
+                ],
+                OUTPUT_CLUSTERS: [
+                    Identify.cluster_id,
+                    OnOff.cluster_id,
+                    Ota.cluster_id,
+                    OppleCluster,
```

Comment:
Is the `OppleCluster` really present on `INPUT_CLUSTERS` and `OUTPUT_CLUSTERS`?
So, in the "Manage Zigbee device" -> Clusters menu, does reading the `mode` attribute on both the "in" and "out" `OppleCluster` work?

(need to press the button whilst reading the `mode` attribute)

Reply:
only the opplecluster on the output seems to be for the battery the rest works